jPaginate.js

```diff
@@ -0,0 +1,207 @@
+// jPaginate Plugin for jQuery - Version 0.3
```

Comment:
can you include this script in the script header? 
like below??
'''
<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
'''